pkg/volume/awsebs/aws_ebs.go

```diff
@@ -250,8 +250,8 @@ func getVolumeSource(
 
 func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath string) (*volume.Spec, error) {
 	mounter := plugin.host.GetMounter(plugin.GetPluginName())
-	pluginDir := plugin.host.GetPluginDir(plugin.GetPluginName())
```

Comment:
is the old `plugin.host.GetPluginDir` func removed now? If not, I would suggest keep the original code, make this PR as small as possible, easy for review.

Reply:
I may move the `path.Join` to `filepath.Join` code to a separate PR. I don't know what happened last night, but my late-night rebase squashed my two commits into one (and I know I didn't do that on purpose).

With that stuff taken out, it's pretty small.

To your specific question, no `GetPluginDir` is not removed. In fact is called from within the new `makeGlobalPDPath()`.